src/cfnlint/rules/functions/Sub.py

```diff
@@ -101,22 +102,12 @@ def _test_parameter(self, parameter, cfn, parameters, tree):
         matches = []
         get_atts = cfn.get_valid_getatts()
 
-        valid_pseudo_params = [
-            'AWS::AccountId',
-            'AWS::NotificationARNs',
-            'AWS::Partition',
-            'AWS::Region',
-            'AWS::StackId',
-            'AWS::StackName',
-            'AWS::URLSuffix',
-        ]
-
         odd_list_params = [
             'CommaDelimitedList',
             'AWS::SSM::Parameter::Value<CommaDelimitedList>',
         ]
 
-        valid_params = valid_pseudo_params
+        valid_params = PSEUDOPARAMS
```

Comment:
You need to switch this to `valid_params = copy(PSEUDOPARAMS)` you are getting hit by a Python thing here.

At the top.
`from copy import copy`

Reply:
Good catch